packages/frontend/app/lib/apollo.server.ts

```diff
@@ -26,50 +23,52 @@ BigInt.prototype.toJSON = function (this: bigint) {
   return this.toString()
 }
 
-const authLink = setContext((request, { headers }) => {
-  if (!process.env.SIGNATURE_SECRET || !process.env.SIGNATURE_VERSION)
-    return { headers }
-  const timestamp = Math.round(new Date().getTime() / 1000)
-  const version = process.env.SIGNATURE_VERSION
+async function createAuthLink(request: Request) {
+  return setContext(async (gqlRequest, { headers }) => {
+    const timestamp = Math.round(new Date().getTime() / 1000)
+    const version = 1
```

Comment:
Wasn't entirely sure what to do about this version. I think we either have to hardcode it like this or make it a configurable at the tenant level. This used to be hardcoded as an env var along with the signature but of course the signature is now set per tenant. 

Im not entirely clear on what the version does and when you'd update it. But I imagine it would be updated alongside a new secret and so it wouldnt make sense to configure this via environment variable while the signature is set per tenant. @njlie do you think hardcoding the version like this makes sense or is there a better way?

Reply:
decided to keep the env var after talking w/ Nathan. The backend has this env var and frontend should match it, so it needs to stay configurable.